### DISCUSSION TITLE
Joined community

### DISCUSSION THREAD

**Discussion by Knightking-9274** [Cyfrin/foundry-full-course-cu#2321] [General]
Hi all, Started with Solidity and Ethereum course on YouTube. Glad to be part of the team!

**Comment by EngrPips** [comment]
Hello @Knightking-9274, welcome aboard and we are very glad to have you.

**Reply by EngrPips** [reply]
Welcome aboard

**Comment by Obies** [comment]
Hi guys glad to be part of the web3 community as I'm learning the course, stay awesome

**Reply by EngrPips** [reply]
Welcome aboard, Friend

**Reply by Dogster83** [reply]
Hi everyone,

Started this week with Solidity and Ethereum course. Thx verry much for creating the course it will help me alot in the near future.

**Reply by EngrPips** [reply]
Welcome aboard @Dogster83, I wish you the best on your journey
